app/Console/Commands/SendAcceptanceReminder.php

```diff
@@ -86,6 +90,11 @@ public function handle()
         }
 
         $this->info($count.' users notified.');
+        $output = "The following users do not have an email address:\n";
+        foreach ($no_email_list as $user) {
+            $output .= "ID: {$user['id']}, Name: {$user['name']}\n";
+        }
+        $this->info($output);
```

Comment:
Using a [table](https://laravel.com/docs/11.x/artisan#tables) here would result in better output.